src/blocks/stars.css

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
```

Comment:
Ensure that the `star.svg` file exists in the `../images/` directory relative to this CSS file. This is crucial for the background image to display correctly.